### DISCUSSION TITLE
How to change the default load directory "/_next"

### DISCUSSION THREAD

**Discussion by Xyn19961118** [vercel/next.js#39021] [Help]
### Describe the feature you'd like to request

Change default is output directory '/_ next/'

### Describe the solution you'd like

Now I have a problem. I have two next projects, which need to be forwarded in nginx on the same project, but CSS and JS are both/_ Under the next/ directory, I want to change one of them/_ Next/, but I don't know what to do? I tried to modify'distdir'and'generatebuildfid' in next.config.js, but it didn't work

### Describe alternatives you've considered

I tried to modify'distdir'and'generatebuildfid' in next.config.js, but it didn't work

**Comment by NicolasSacC** [comment]
Did you find a solution for that?
